cypress/integration/inventoryPage.js

```diff
@@ -0,0 +1,43 @@
+const {it} = require('mocha');
```

Comment:
class naming: if the page contains tests, it should be called like inventoryTests or inventory.tests

Reply:
Fixed

Reply:
Fixed